service/src/main/java/gov/nasa/pds/api/registry/model/RefLogicCollection.java

```diff
@@ -11,20 +11,17 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import gov.nasa.pds.api.registry.*;
```

Comment:
Wildcard imports 🤮

Reply:
Whoops, that was some "helpful" IDE intervention that I should've caught.  Will fix (tyvm for the catch!)